src/Main.js

```diff
@@ -5,80 +5,54 @@ import List from './List'
 import movieData from './mockData'
 import './Main.css'
 
-// 👇 hide - show using state and re-rendering completely
 class Main extends React.Component {
   constructor() {
     super();
     this.state = {
       movieData: movieData.movies,
-      //classes: true
+      isExtendedView: false,
+      clickedPosterID: null
     }
   }
 
+  changeExtendedState = (id) => {
+    this.handlePosterClick(id)
 
-  // hidePostersView(event) {
-  //
-  //     this.setState({ classes: })
-  //     event.target.parentElement.parentNode.classList.add('hidden')
-  //     //show specific ExtendedView
-  //
-  //
-  // }
-
-  // render() {
-  //       if(this.state.classes) {
-  //           return (
-  //             <Header />
-  //             <Posters />
-  //             <Footer />
-  //           )
-  //       } else {
-  //           return (
-  //             <Header />
-  //             <ExtendedView id={event.target.id}/>
-  //             <Footer />
-  //           )
-  //       }
-  //   }
-//state starts out true (only render Posters component)
-//on poster click, changes state property to false
-
-//first function: hideShow() - this will toggle the hidden class
-
-//second function: showExtendedMovieView() -
-//1. invoked when user clicks on movie
-//2. grabs id of event.target
-//2. this will fire hideShow()
-
+    if (this.state.isExtendedView) {
```

Comment:
Possibly make this an if else statement to eliminate to if statements? (ie. if (condition) "do this", else "do that"). Just a thought!